src/arch/arch_default.h

```diff
@@ -57,22 +89,22 @@
 #include "MemoryCard/MemoryCardDriver_Null.h"
 #include "MemoryCard/MemoryCardDriverThreaded_Folder.h"
 
-#endif
+#endif // ARCH_DEFAULT_H
 
 /*
  * (c) 2002-2006 Glenn Maynard, Ben Anderson, Steve Checkoway
  * All rights reserved.
- * 
+ *
  * Permission is hereby granted, free of charge, to any person obtaining a
  * copy of this software and associated documentation files (the
  * "Software"), to deal in the Software without restriction, including
  * without limitation the rights to use, copy, modify, merge, publish,
  * distribute, and/or sell copies of the Software, and to permit persons to
  * whom the Software is furnished to do so, provided that the above
  * copyright notice(s) and this permission notice appear in all copies of
- * the Software and that both the above copyright notice(s) and this
```

Comment:
For now let's keep these line breaks the same

Reply:
my mistake